namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -46,20 +49,30 @@ private[consul] object SvcAddr {
     preferServiceAddress: Option[Boolean] = None,
     tagWeights: Map[String, Double] = Map.empty,
     stats: Stats,
-    stateWatch: v1.PollState[String, v1.IndexedServiceNodes]
+    stateWatch: v1.PollState[String, v1.IndexedServiceNodes],
+    transferMetaData: Option[Boolean] = None
```

Comment:
Let's make this just a `Boolean` throughout the consul namer and library code.  The ConsulInitializer should be responsible for taking the `Option[Boolean]` from the config and applying a default.